runtime/src/asset.rs

```diff
@@ -1475,9 +1520,19 @@ impl<T: Trait> Module<T> {
         Self::_update_checkpoint(ticker, to_did, current_to_balance);
 
         <BalanceOf<T>>::insert(&ticker_to_did, updated_to_balance);
-        <Tokens<T>>::insert(ticker, token);
-
-        Self::deposit_event(RawEvent::Issued(ticker.clone(), to_did, value));
+        <Tokens<T>>::insert(ticker.clone(), token);
+        let round = Self::funding_round(ticker);
+        let issued_in_this_round = Self::issued_in_funding_round((ticker.clone(), round.clone()))
```

Comment:
Ditto: `mutate` instead of `get` + `insert`.

Reply:
Same [observation](https://github.com/PolymathNetwork/Polymesh/pull/169#discussion_r369526726) applies here.